ClientApp/src/setupProxy.js

```diff
@@ -6,7 +6,8 @@ const target = env.ASPNETCORE_HTTPS_PORT ? `https://localhost:${env.ASPNETCORE_H
 
 const context =  [
   "/weatherforecast",
-  "/helloback"
+  "/helloback",
+  "/login"
```

Comment:
There is already a React route to this link. For communication with the backend please set up a different route. Currently when trying to access /login user gets 404

Reply:
Fixed (hopefully)